include/swift/Serialization/ModuleFormat.h

```diff
@@ -55,7 +55,7 @@ const uint16_t VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t VERSION_MINOR = 427; // SWIFT_ENABLE_TENSORFLOW: serialize @differentiable.
+const uint16_t VERSION_MINOR = 428; // SWIFT_ENABLE_TENSORFLOW: ParameterAggregate.
```

Comment:
I don't think anything changed the module format.

Reply:
Reverted in https://github.com/apple/swift/pull/18140/commits/7768fa9d6d142cf111f4f110d44b68dd1cf52a74.